### DISCUSSION TITLE
Getting '404 page not found' error while using Artifactory as proxy for the GitHub repository where my Maven package is published.

### DISCUSSION THREAD

**Discussion by Sekhar9581** [community/community#49939] [Packages]
### Select Topic Area

Question

### Body

I have published Maven package into GitHub repository and Im trying to use Artifactory as an proxy to fetch the Maven packages from GitHub repository. For this, I have created one remote repository in Artifactory and configured with GitHub repo URL, I have set Repository layout and remote layout mapping to 'maven-2-default'.. And for authentication, I have entered my GitHub username and PAT(with write permission to packages) in remote authentication section. After configuring all settings in Artifactory, I clicked on test icon to test the connection, then I see "Connection failed: Error 404: Not Found" error message. I have verified that URL and PAT are valid but still getting 404 error
Looking for the help to fix this 404 error and make successful connection.

**Comment by Sekhar9581** [comment]
Issue got fixed

**Reply by sjl0825** [reply]
Hi @Sekhar9581,
How did you fix this issue?
Thank you!
